src/managers/input/InputManager.cpp

```diff
@@ -87,6 +87,11 @@ void CInputManager::onMouseMoved(IPointer::SMotionEvent e) {
 
     const auto  DELTA = *PNOACCEL == 1 ? e.unaccel : e.delta;
 
+    if (g_pSeatManager->isPointerFrameSkipped)
```

Comment:
am I blind or are the stored values never used

Reply:
listener_commitWindow sends them

Reply:
right.